.eslintrc

```diff
@@ -30,6 +30,7 @@
     "node": true
   },
   "rules": {
-    "no-unused-vars": 1
+    "no-unused-vars": 1,
+    "react/no-find-dom-node": 0
```

Comment:
Unless I'm missing a better way to do our layout, I think we'll need to use `findDOMNode()` (though infrequently).